SecretSanta/src/SecretSanta.Business/Gift.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Business
+{
+    public class Gift
+    {
+        private int _Id;
+        private string _Title;
+        private string _Description;
+        private string _Url;
+        private User _User;
+
+#pragma warning disable CA1054 // Uri parameters should not be strings
```

Comment:
```suggestion
#pragma warning disable CA1054 // Uri parameters should not be strings
```

Maybe move this to a global suppression file?